wpi.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+# This script performs WPI via dljc on a given project directory.
+# The inputs are similar to wpi-many.sh, which uses this script internally.
+# The only difference is that wpi-many.sh takes a list of projects, while
+# this script operates on a single project at a time.
+# See the documentation of wpi-many.sh for information on the inputs to this
+# script.
+#
+# Input differences compared to wpi-many.sh:
+# -i and -o are not valid options
+# new required option -d: the absolute path to the directory containing the target project
+#
+
+while getopts "d:u:t:" opt; do
+  case $opt in
+    d) DIR="$OPTARG"
+       ;;
+    u) USER="$OPTARG"
+       ;;
+    t) TIMEOUT="$OPTARG"
```

Comment:
Is it useful to run the scripts with no timeout?  If not, I suggest setting a default.

Reply:
Not unless we figure out how to deal with the infinite descending chains problem. But, if we're planning to work on that at some point in the future, we would need to be able to run with no timeout. So I don't see a reason to remove the ability to do so.